connect-widget/app/src/shared/components/WidgetContainer.tsx

```diff
@@ -56,6 +57,14 @@ const WidgetContainer = ({ children }: WidgetContainerProps) => {
     },
   });
 
+  const [scrolled, setScrolled] = useState(false);
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const handleScroll = (event: any) => {
+    if (event.currentTarget.scrollTop > 120) setScrolled(true);
```

Comment:
why are we hardcoding `120`? should we use a media query here?

Reply:
I'll change it to a fraction of the window.innerHeight